index.html

```diff
@@ -207,17 +207,20 @@ <h4>Skills</h4>
                         </ul>
                     </div>
                 </li>
-            </ul>
-            
+            </ul>        
+        </section>
+
+        <section id="contact-form">
             <div class="form-info">
                 <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
             </div>
 
-            <input type="text" class="input" placeholder="Full name">
-            <input type="email" class="input" placeholder="Email adress">
-            <textarea name="" id="" cols="30" rows="10" class="input textarea" placeholder="Enter text here"></textarea>
-            
-            <a href="#" class="button">Get in touch</a>
+            <form action="">
```

Comment:
- [ ] As per the project requirement, kindly make sure you send the request to the form spree server to get the form to your email when the user submit :recycle:, [here](https://formspree.io/html/) is the way you can do it :thought_balloon: